HarmonyPatches/HookLevelCollectionTableSet.cs

```diff
@@ -113,7 +113,7 @@ static class HookLevelCollectionTableSet {
 				previewBeatmapLevels = outV.ToArray();
 
 				if(sorter is ISorterWithLegend sl && Config.Instance.EnableAlphabetScrollbar)
-					customLegend = sl.BuildLegend(previewBeatmapLevels).ToArray();
+					customLegend = sl.BuildLegend(previewBeatmapLevels.ToArray()).ToArray();
```

Comment:
Why did you call `ToArray` here? It is already.

Reply:
Good catch but when its an array alreads its essential NOOP so doesnt matter, I'll fix it tho when I release

Reply:
It is an `IReadOnlyList<BeatmapLevel>` not an array.
![image](https://github.com/kinsi55/BeatSaber_BetterSongList/assets/31044781/0ef5bb44-d2f0-44f4-9d8f-d967fca7d11c)

Reply:
An array is a `IReadOnlyList<BeatmapLevel>`. You have to cast it to `BeatmapLevel[]`.

Reply:
I forked your fork and opened a PR that contains this cast.